lambda/file_format.tf

```diff
@@ -0,0 +1,181 @@
+resource "aws_lambda_function" "file_format_lambda_function" {
+  count         = local.count_file_format
+  function_name = "${var.project}-file-format-${local.environment}"
+  handler       = "uk.gov.nationalarchives.fileformat.Lambda::process"
+  role          = aws_iam_role.file_format_lambda_iam_role.*.arn[0]
+  runtime       = "java8"
+  filename      = "${path.module}/functions/file-format.jar"
+  timeout       = 180
+  memory_size   = 1024
+  tags          = var.common_tags
+  environment {
+    variables = {
+      ENVIRONMENT   = local.environment
+      INPUT_QUEUE   = local.file_format_queue_url
+      OUTPUT_QUEUE  = local.api_update_queue_url
+      AUTH_URL      = var.auth_url
+      API_URL       = "${var.api_url}/graphql"
+      CLIENT_ID     = "tdr-backend-checks"
```

Comment:
Should the CLIENT_ID value be hardcoded?

Reply:
I think it's fine. Because it's defined in the keycloak repo we'd have to change it in here even if we put it in the parameter store and it's not secret so it's ok in the repos.

Reply:
OK.